do-not-commit/use-timeout.py

```diff
@@ -0,0 +1,32 @@
+import requests
+
+url = "www.github.com"
+
+# ruleid: use-timeout
+r = requests.get(url)
+
+# ruleid: use-timeout
+r = requests.post(url)
+
+# ok
+r = requests.get(url, timeout=50)
+
+
+def from_import_test1(url):
+    from requests import get, post
+
+    # ok
+    r = get(url, timeout=3)
+
+    # ruleid: use-timeout
+    r = post(url)
```

Comment:
By default, 'requests' calls wait until the connection is closed.
This means a 'requests' call without a timeout will hang the program
if a response is never received. Consider setting a timeout for all
'requests'. ([actice.use-timeout.use-timeout](https://semgrep.dev/r/actice.use-timeout.use-timeout))
#
<div>
<a href="https://semgrep.dev">
<img align="right" src="https://r2c.dev/semgrep_icon_text.svg" height=28px alt="" />
</a>

<sub> [How do I suppress this?](https://semgrep.dev/docs/running-rules/#managing-findings) **·** [Report a bug](https://github.com/returntocorp/semgrep-rules/issues/new?assignees=&labels=&template=rule_bug_report.md&title=actice.use-timeout.use-timeout) **·** [Manage rule](https://semgrep.dev/manage/projects)</sub>
</div>